Source/WebCore/page/DeprecatedGlobalSettings.h

```diff
@@ -170,7 +170,7 @@ class DeprecatedGlobalSettings {
 #endif
 
 #if HAVE(AVCONTENTKEYSPECIFIER)
-    static void setSampleBufferContentKeySessionSupportEnabled(bool);
+    WEBCORE_EXPORT static void setSampleBufferContentKeySessionSupportEnabled(bool);
```

Comment:
Is there a reason to restore this? Adding `webcorebinding: DeprecatedGlobalSettings` doesn't require `WEBCORE_EXPORT`

Reply:
Yes it does -- see the build failure from the previous version of this patch.

Reply:
Ah https://searchfox.org/wubkat/rev/a19946e70e2fc15526742690123bc3f4afc4ff0f/Source/WebKit/GPUProcess/GPUProcessPreferences.cpp#75 is what uses it outside of WebCore.

Reply:
```
ld: warning: building for iOS, but linking in .tbd file (/Volumes/Data/worker/iOS-16-Build-EWS/build/WebKitLibraries/WebKitPrivateFrameworkStubs/iOS/16/GraphicsServices.framework/GraphicsServices.tbd) built for iOS Simulator
Undefined symbols for architecture arm64:
  "WebCore::DeprecatedGlobalSettings::setSampleBufferContentKeySessionSupportEnabled(bool)", referenced from:
      -[WebView(WebViewInternalPreferencesChangedGenerated) _preferencesChangedGenerated:] in WebViewPreferencesChangedGenerated.o
ld: symbol(s) not found for architecture arm64
clang: error: linker command failed with exit code 1 (use -v to see invocation)
```

I think if you remove WEBCORE_EXPORT from anything actually used by the WebKit or WebKitLegacy layers you will get this build error.